tests/app/test_saver.py

```diff
@@ -82,6 +82,46 @@ def test_save_msg_audit_raises_message_save_exception_on_db_error(self):
                 with self.assertRaises(MessageSaveException):
                     Saver().save_msg_audit(message_audit['msg_id'], message_audit['msg_urn'], mock_session)
 
+    def test_save_actors_does_not_raise_exception_on_successful_save(self):
+        """Tests message actors is saved to database"""
+        message = {'msg_id': 'MsgId', 'msg_to': 'Tej', 'msg_from': 'some survey'}
+        with self.app.app_context():
+            with current_app.test_request_context():
+                Saver().save_message(SecureMessage(msg_id=message['msg_id']))
+                Saver().save_msg_actors(message['msg_id'], message['msg_to'], message['msg_from'], False)
+                self.assertTrue(True)
+
+    def test_save_actors_raises_message_save_exception_on_db_error(self):
+        """Tests MessageSaveException generated if db commit fails saving message actors"""
+        mock_session = mock.Mock(db.session)
+        mock_session.commit.side_effect = Exception("Not Saved")
+        message = {'msg_id': 'MsgId', 'msg_to': 'Tej', 'msg_from': 'someone'}
+        with self.app.app_context():
+            with current_app.test_request_context():
+                with self.assertRaises(MessageSaveException):
+                    Saver().save_msg_actors(message['msg_id'], message['msg_to'], message['msg_from'], False, mock_session)
+
+    def test_save_actors_saves_expected_information(self):
+        """Simple read write validation of actor table"""
+        pass
```

Comment:
Shouldn't be here?

Reply:
I know that there is an argument for not doing this sort of tests if you view it as testing the db. However It does test the __init__ and the set from domain model code in the Actors class .

Reply:
I mean the `pass` rather than the test itself :)

Reply:
Just shoot me now. Changing ( with a red face)